src/syntax/ppx_sedlex.ml

```diff
@@ -489,9 +667,13 @@ let mapper =
         let l, regexps' = sub#structure_with_regexps (!previous @ l) in
         let parts = List.map partition (get_partitions ()) in
         let tables = List.map table (get_tables ()) in
+        let funcs =
+          let loc = default_loc in
+          [%str let __sedlex_tl = function _ :: tl -> tl | _ -> assert false]
```

Comment:
Can we put this in the lib, similar to what we do for `Sedlexing.__private__next_int`

Reply:
This comment is not relevant anymore